core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2061,9 +2061,8 @@ class SparkContext(config: SparkConf) extends Logging {
    * Cancel a given job if it's scheduled or running.
    *
    * @param jobId the job ID to cancel
-   * @throws InterruptedException if the cancel message cannot be sent
+   * @note Throws `InterruptedException` if the cancel message cannot be sent
```

Comment:
Hm, InterruptedException is in `java.lang` so I am surprised it isn't found. Does it help if you write `@throws java.lang.InterruptedException`? that's better if it works